events/generate/data/firebase-db1-cloudevent-input.json

```diff
@@ -0,0 +1,14 @@
+{
+  "specversion": "1.0",
+  "type": "google.firebase.database.document.v1.written",
+  "source": "//firebase.googleapis.com/projects/_/instances/my-project-id/refs/gcf-test/xyz",
+  "id": "aaaaaa-1111-bbbb-2222-cccccccccccc",
+  "time": "2020-05-26T10:42:27.088Z",
+  "datacontenttype": "application/json",
+  "data": {
+    "data": null,
```

Comment:
Can we provide non-null data to verify this input works appropriately?

Reply:
I just used the existing firebase-db1 event -- if you have data that you would suggest, I can replace it.

Reply:
This works, nvm.